src/modules/85-cv/components/ExecutionVerification/components/ExecutionVerificationStepShortView/__tests__/ExecutionVerificationStepShortView.test.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react'
+import { render, screen } from '@testing-library/react'
+import userEvent from '@testing-library/user-event'
+import { act } from 'react-dom/test-utils'
+import { TestWrapper } from '@common/utils/testUtils'
+import type { ExecutionNode } from 'services/pipeline-ng'
+import ExecutionVerificationStepShortView from '../ExecutionVerificationStepShortView'
+import { executionMetadata, stepDetails, stepDetailsWithoutPolicy } from './ExecutionVerificationStepShortView.mock'
+
+jest.mock('../../ExecutionVerificationSummary/ExecutionVerificationSummary', () => {
+  return { ExecutionVerificationSummary: () => <div data-testid="ExecutionVerificationSummary" /> }
+})
+
+describe('ExecutionVerificationStepShortView', () => {
+  test('ExecutionVerificationStepShortView should render the right policy details', () => {
+    const { container } = render(
+      <TestWrapper>
+        <ExecutionVerificationStepShortView
+          step={stepDetails as unknown as ExecutionNode}
+          executionMetadata={executionMetadata}
+        />
+      </TestWrapper>
+    )
+
+    expect(screen.getByTestId(/ExecutionVerificationSummary/)).toBeInTheDocument()
+    expect(container.querySelector('[data-tab-id="POLICY_ENFORCEMENT"]')).toBeInTheDocument()
```

Comment:
querySelector is not recommended to query by class or id because they are invisible to the user, you can use getByRole() to query every element that is exposed in the accessibility tree if possible.

```suggestion
   expect(screen.getByRole('tab', {
      name: 'pipeline.policyEnforcement.title'
  })).toBeInTheDocument()
```
https://kentcdodds.com/blog/common-mistakes-with-react-testing-library

Reply:
Updated @Eric761